jupyterlab/commands.py

```diff
@@ -34,7 +34,10 @@ def run(cmd, **kwargs):
     print('> ' + list2cmdline(cmd))
     kwargs.setdefault('shell', sys.platform == 'win32')
     kwargs.setdefault('env', os.environ)
-    return check_output(cmd, **kwargs)
+    try:
+        return check_output(cmd, **kwargs)
+    except CalledProcessError as error:
+        raise error
```

Comment:
what error are you referring to? I saw the notice about git describe throwing an error - that one?